README.md

```diff
@@ -31,6 +31,20 @@ NLU_URL=<NLU_URL>
 NLU_API_KEY=<NLU_API_KEY>
 ```
 
+In the same `.env` file, add a comma separated list of origins that will be allowed to make cross site requests.
+
+If any origin can make the requests, just use the "*" value.
+
+```
+ALLOWED_ORIGINS=<ALLOWED_ORIGINS>
+```
+
+For example:
+```
+ALLOWED_ORIGINS=http://localhost:3001
+```
+
+
```

Comment:
Instead of setting `ALLOWED_ORIGINS` as mandatory, can we set it to `http://localhost:3001` by default and move the update instruction to the _Customize_ section bellow?